### PR TITLE
Add prettify to validate_indices

### PR DESCRIPTION
**Pull request by WackerO**
ATM, a call to `prettifyVariablename` is missing in `validate_indices` while `stringsToNamedVector` contains a prettify. This leads to `exploratory_plots.R` throwing an `error Invalid` assays because it will e.g. compare the strings `raw` and `Raw`. This PR adds the prettify to fix this issue.

### CODE REVIEW

File: R/accessory.R [hunk 1]
```diff
@@ -1271,6 +1271,7 @@ validate_indices <- function(assay_data, index_string) {
     indices <- as.integer(simpleSplit(index_string))
   } else {
     indices <- simpleSplit(index_string)
+    indices <- lapply(indices, prettifyVariablename)
```
Comment (pinin4fjords) [inline R/accessory.R:1274]: You'll need an unlist here to maintain the same type as simpleSplit.

**Review by pinin4fjords — CHANGES_REQUESTED** [review]
Sorry, just looking at this properly. 

You'll notice that stringsToNamedVector has the prettification as a parameter. It should probably be the same in this function (with the same default), so that things can be matched.

**Comment by WackerO** [comment]
> Sorry, just looking at this properly.
> 
> You'll notice that stringsToNamedVector has the prettification as a parameter. It should probably be the same in this function (with the same default), so that things can be matched.

Ah yes sure, done!

**Review by pinin4fjords — APPROVED** [review]
LGTM

**Pull request merged**